### PR TITLE
[test-only]changed sharing test

### PR DESCRIPTION
**Pull request by ScharfViktor**
part of https://github.com/owncloud/web/issues/10259

- [x] deleted acceptance tests related sharing actions
- [x] added missed e2e test for:
- - [x] remove share with group
- - [x] sharing using quick action
- - [x] update share using custum permissions
- [x] copy/move test. expect 201 and 204 code responses   
- [x] shared with others - check displaying resources with same name

### CODE REVIEW

File: tests/e2e/cucumber/features/smoke/copyMove.feature [hunk 1]
```diff
@@ -175,13 +175,11 @@ Feature: Copy
...
-    # | example1.txt | folder1 | replace   |
+      | example1.txt | folder1 | replace   |
     And "Alice" moves the following resource using sidebar-panel
-      | resource     | to          | option    |
-      | example1.txt | sub/folder1 | keep both |
-    # issue https://github.com/owncloud/web/issues/10515
-    # | folder1/example1.txt | sub/folder1 | replace   |
+      | resource             | to          | option    |
+      | example1.txt         | sub/folder1 | keep both |
+      | folder1/example1.txt | sub/folder1 | replace   |
```
Comment (saw-jan) [inline tests/e2e/cucumber/features/smoke/copyMove.feature:182]: was the issue fixed?

**Review by saw-jan — APPROVED** [review]
Others look good. Also please, backport it to the stable-8

File: tests/e2e/cucumber/features/smoke/copyMove.feature (hunk 1 above)
Comment (ScharfViktor) [reply tests/e2e/cucumber/features/smoke/copyMove.feature:182]: no, issue is relevant only for copying moving folder

for file works correct

**Comment by ScharfViktor** [comment]
> Also please, backport it to the stable-8

https://github.com/owncloud/web/pull/10543

**Pull request merged**